scripts/rebuild-deb.sh

```diff
@@ -113,8 +113,8 @@ cat "${BUILDDIR}/DEBIAN/control"
 echo "------------------------------------------------------------"
 # Binaries
 mkdir -p "${BUILDDIR}/usr/local/bin"
```

Comment:
here's your problem, the mina package has to actually include a mina binary. This should be `sudo cp ./default/src/app/cli/src/mina.exe "${BUILDDIR}/usr/local/bin/mina"` and the same for mina-rosetta